contracts/test/DaiEchidnaTest.sol

```diff
@@ -42,25 +48,28 @@ contract DaiEchidnaTest {
     /// @dev Test that supply and balance hold on burn
     function burn(uint256 wad) public {
         uint256 supply = dai.totalSupply();
-        uint256 holderBalance = dai.balanceOf(holder);
+        uint256 holderBalance = dai.balanceOf(holder) == 0 ? 1 : dai.balanceOf(holder);
+        wad = 1 + wad % holderBalance;
```

Comment:
I think this still isn't quite right--if `dai.balanceOf(holder) == 0`, you can't `burn` any amount other than zero. So I'd do something more like:

```
holderBalance = dai.balanceOf(holder);
wad = holderBalance == 0 ? 0 : 1 + wad % holderBalance
```

Reply:
Done.